src/components/Home.js

```diff
@@ -140,6 +143,45 @@ const Home = ({ classes }) => {
           </JsonFormsReduxContext>
         </Provider>
       </div>
+
+      <div className={styles.landing_page__architecture}>
+        <Grid container spacing={3} alignItems="center">
+          <Grid item xs={12} md={8}>
+            <Typography variant='h3' align='center' className={styles.shift_left}>JSON Forms Architecture</Typography>
+          </Grid>
+        </Grid>
+        <Grid container spacing={3} alignItems="center">
+          <Grid item xs={12} md={8}>
+            <img src={architectureDefault} alt="JSON Forms Architecture" className={styles.architecture_default} />
+          </Grid>
+          <Grid item xs={12} md={4}>
+            <Paper className={styles.architecture_description}>
+              <p>
+                The basis of JSON Forms is the core package (<code>@jsonforms/core</code>) which provides utilities for managing and rendering JSON Schema based forms. The core packages is independent of any UI technology.
+                <br/><br/>
+                We also provide the JSON Forms React (<code>@jsonforms/react</code>) and JSON Forms Angular (<code>@jsonforms/angular</code>) packages. These use the core package to provide specialized bindings for React and Angular. This approach is especially useful when developing multiple renderer sets against the same technology (i.e. React) as the core bindings don't need to be reimplemented with each set.
+                <br/><br/>
+                For React we maintain two renderer sets: The <code>@jsonforms/material-renderers</code>, which are based on the popular <a href="https://material-ui.com/">Material-UI</a> framework. The second package <code>vanilla-renderers</code> provides pure HTML5 renderers. For Angular we provide a <a href="https://material.angular.io/">Angular Material</a> based renderer set.
+                <br/><br/>
+                We lay great importance on the customizability and extensibility of JSON Forms. Not only are the existing renderers declaritively configurable, you can also overwrite any of them with your own custom renderers for advanced use cases. Even when you would like to use a different UI framework (e.g. Bootstrap) you can still reuse the JSON Forms core and React / Angular packages to help you on the way.
+              </p>
+            </Paper>
+          </Grid>
+        </Grid>
+        <Grid container spacing={3} alignItems="center">
+          <Grid item xs={12} md={8}>
+            <img src={architectureCustom} alt="JSON Forms Architecture" className={[styles.architecture_custom, styles.shift_left].join(' ')} />
+          </Grid>
+          <Grid item xs={12} md={4}>
+            <Paper className={styles.architecture_description}>
+              <p>
+                Instead of using the provided binding frameworks, you can also create your own binding framework. For this framework you can then create your own renderer set.
```

Comment:
```suggestion
                In case you would like a different application framework you could even create the bindings yourself. In this case you still get use out of JSON Forms via the core package.
```